Rubberduck.Refactorings/MoveCloserToUsage/MoveCloserToUsageModel.cs

```diff
@@ -1,14 +1,27 @@
-﻿using Rubberduck.Parsing.Symbols;
+﻿using Rubberduck.Parsing.Grammar;
+using Rubberduck.Parsing.Symbols;
 
 namespace Rubberduck.Refactorings.MoveCloserToUsage
 {
     public class MoveCloserToUsageModel : IRefactoringModel
     {
-        public Declaration Target { get; }
+        private Declaration _target;
+        public Declaration Target
+        {
+            get => _target;
+            set
+            {
+                _target = value;
+            }
+        }
+
+
+        public string DeclarationStatement { get; set; } = string.Empty;
 
```

Comment:
Initializing `DeclarationStatement` to `Tokens.Static` would be wrong if a 'Dim' declaration inside a method gets moved, because that must remain as 'Dim'. Therefore the the `GetDefaultDeclarationStatement` Method is used as descriped above.

I don't know if renaming to `Accessibility` isn't missleading. When it's containing `Public`, `Private`, `Static` it's ok but for `Dim ` the Name `Accessibility` doesn't really fit for me.

At the moment there are two Options. While writing i encountered a third option (move between modules from public to private) which dosn't work correctly at the moment. So i would stay with the `String `-solution.